BOPIS/BOPIS/View/LoginView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  LoginView.swift
+//  BOPIS
+//
+//  Created by Ganesh Ubale on 06/10/22.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    @State var userName:String = ""
+    @State var password:String = ""
```

Comment:
Can we move username and  password variables inside view model? Also, we can create a constant called EMPTY_STRING in constant file and access from there instead of double quotes.

Reply:
Added emptyString constant as you suggest, we will move those userName and password when we create ViewModel. for now I added comment to remind this.